src/seedu/addressbook/data/person/Address.java

```diff
@@ -50,10 +45,9 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        return value.hashCode();
+        return super.hashCode();
```

Comment:
if you are not doing anything special in the subclass there is no need to call the super.X(). If the subclass does not override a method, automatically the superclass method will be called.